addons/event/static/src/js/tests/online_event_groupby_tests.js

```diff
@@ -0,0 +1,76 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+
+import { getFixture } from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+
+let target;
+
+QUnit.module("Event", (hooks) => {
+    hooks.beforeEach(async () => {
+        const pyEnv = await startServer();
+        pyEnv.mockServer.models["event.event"] = {
+            fields: {
+                id: { string: "Id", type: "integer" },
+                name: { string: "Event Template", type: "char" },
+                address_id: {
+                    string: "Venue",
+                    type: "many2one",
+                    relation: "res.partner",
+                },
+            },
+            records: [
+                {
+                    id: 1,
+                    name: "My event",
+                    address_id: false,
+                },
+            ],
+        };
+        target = getFixture();
+        setupViewRegistries();
+    });
+    QUnit.test("Test group label for online", async function (assert) {
+        assert.expect(1);
+        
+        const views = {
+            "event.event,false,list":
+                `<tree js_class="event_list_views"/>`,
+            "event.event,false,search": `<search/>`,
+        };
+        const { openView } = await start({
+            serverData: { views },
+        });
+        await openView({
+            res_model: "event.event",
+            views: [[false, "tree"]],
+            context: { group_by: ["address_id"] },
+        });
+
+        assert.strictEqual(target.querySelector(".o_group_name").innerText, "Online (1)");
+    });
+    QUnit.test("Test group label for online event", async function (assert) {
```

Comment:
`... for online event in kanban view` and you can move the kanban test before the tree one to keep the alphabetical order 🙂